pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "docling-core"
-version = "2.48.4"  # DO NOT EDIT, updated automatically
+version = "2.43.0"  # DO NOT EDIT, updated automatically
```

As written on the comment, this should not be edited; version gets bumped by CD pipeline upon release.
```suggestion
version = "2.48.4"  # DO NOT EDIT, updated automatically
```